pyEpiabm/pyEpiabm/tests/test_unit/test_routine/test_file_population_config.py

```diff
@@ -217,8 +218,13 @@ def test_print_population(self, mock_copy, mock_read):
         actual_df = mock_copy.call_args.args[0]
         if version.parse(pd.__version__) >= version.parse("1.4.0"):
             expected_df = self.df.copy()
-            expected = expected_df.drop('household_number', axis=1)
-            actual = actual_df.drop('household_number', axis=1)
+
+            # The type of the id is converted from float to string, so we
+            # drop the cell and microcell columns
```

Comment:
Why is this? We should still check these match, and just convert back on the modified version. Better still, don't modify the data frame in the code, just convert to a string when you set the ID

Reply:
Ah yes, thanks, have made the change